src/main/java/com/suse/salt/netapi/client/impl/HttpClientConnection.java

```diff
@@ -23,9 +22,11 @@
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.http.impl.client.HttpClients;
 
-import java.io.IOException;
-import java.io.UnsupportedEncodingException;
-import java.net.URI;
+import com.suse.salt.netapi.client.Connection;
+import com.suse.salt.netapi.config.ClientConfig;
+import com.suse.salt.netapi.exception.SaltException;
+import com.suse.salt.netapi.exception.SaltUserUnauthorizedException;
+import com.suse.salt.netapi.parser.JsonParser;
```

Comment:
Please don't let your IDE reorganize all imports just for fun, in this case there should be only one additional import (`CookieSpecs`).